spacy_llm/tasks/__init__.py

```diff
@@ -7,6 +11,25 @@
 from .summarization import SummarizationTask, make_summarization_task
 from .textcat import TextCatTask, make_textcat_task
 
+# Register llm_TASK factories with NoOp models.
+for task_handle in registry.llm_tasks.get_all():
+    Language.factory(
+        name=f"llm_{task_handle.split('.')[1].lower()}",
```

Comment:
I think this should be restricted to a manual list of the most recent task from `spacy-llm` with using `get_all`. This parsing is very brittle and would break on some user functions with arbitrary names. I think it would also try to process all versions of tasks, also legacy ones?

Reply:
Fair point. We don't need `get_all()` at all then though, no?

Reply:
Updated to manual list in https://github.com/explosion/spacy-llm/pull/243/commits/64ca71a85c26c5e6f5a3df4f7868484f445d0623 and https://github.com/explosion/spacy-llm/pull/243/commits/50c8f213a68a5ab53170e036214124366eeb0519.